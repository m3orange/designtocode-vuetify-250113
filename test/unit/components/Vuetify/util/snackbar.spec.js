import { test } from '@/test'
import Snackbar from '@/components/Vuetify/util/snackbar'

test('snackbar.js', ({ mount }) => {
  it('should functionally work', async () => {
    const wrapper = mount({
      data: () => ({}),
      render: h => h('div', {
        attrs: {
          'data-app': true,
        },
        staticClass: 'application',
      }, h('div'))
    }, {
      attachToDocument: true,
    })
    wrapper.vm.$snackbar = Snackbar
    wrapper.vm.$snackbar.error({
      message: 'test-message',
      timeout: 1000
    })
    await wrapper.vm.$nextTick()
    expect(wrapper.html()).toMatchSnapshot()
    expect(wrapper.contains('.v-snack')).toEqual(true)
    expect(wrapper.contains('.error')).toEqual(true)
  });
})