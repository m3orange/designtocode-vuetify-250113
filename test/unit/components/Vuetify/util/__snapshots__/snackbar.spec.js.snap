// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snackbar.js should functionally work 1`] = `

<div data-app="true"
     class="application"
>
  <div class="v-snack v-snack--active v-snack--bottom snack-transition-enter snack-transition-enter-active">
    <div class="v-snack__wrapper error">
      <div class="v-snack__content">
        test-message
      </div>
    </div>
  </div>
</div>

`;
